services/tables/src/main/java/com/linkedin/openhouse/tables/dto/mapper/iceberg/PoliciesSpecMapper.java

```diff
@@ -29,7 +29,9 @@ public class PoliciesSpecMapper {
   public String toPoliciesJsonString(TableDto tableDto) throws JsonParseException {
     if (tableDto.getPolicies() != null) {
       try {
-        Gson gson = new GsonBuilder().setPrettyPrinting().create();
+        Gson gson = new GsonBuilder().setPrettyPrinting()
```

Comment:
does it make sense to share this object instead of init in different places with the same config ?

Reply:
make the init builder in a static method to the class